### PR TITLE
Add logic to email over Magic Link

### PR DESCRIPTION
**Pull request by rjvani**

### CODE REVIEW

File: app/services/email_manager/magic_link_sender.rb
Comment (wilsonj806) [inline app/services/email_manager/magic_link_sender.rb:24]: Should add `rel="noopener norefferrer"` to the a tag for best practice/ also for security.

**Review by wilsonj806 — APPROVED** [review]
Looks good to me! Probably should wait for additional approval cuz my Ruby's not amazing.

**Review by jacoblb64 — APPROVED** [review]
Looks good to me!

**Pull request merged**
